{"name": "Проверка невозможности авторизоваться с несуществующим паролем. Получаем ошибку 404", "status": "passed", "steps": [{"name": "Логинимся курьером", "status": "passed", "parameters": [{"name": "login", "value": "'hfamocuzde'"}, {"name": "password", "value": "'hfamocuzde'"}], "start": 1719232035769, "stop": 1719232035916}], "start": 1719232035768, "stop": 1719232035917, "uuid": "05796918-bde4-4e8d-97f6-f6380d813693", "historyId": "0a0764930809d5a8c7173afb9316a6eb", "testCaseId": "0a0764930809d5a8c7173afb9316a6eb", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_with_incorrect_password", "labels": [{"name": "story", "value": "Тесты на вход курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-7KB2K2E"}, {"name": "thread", "value": "15328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}